app/routes/admin/groups/components/GroupMembersList.js

```diff
@@ -45,18 +46,16 @@ const GroupMembersList = ({
 }: Props) => {
   const GroupMembersListColumns = (fullName, membership) => {
     const { user, abakusGroup } = membership;
-    const performRemove = () =>
-      confirm(
-        `Er du sikker på at du vil melde ut "${user.fullName}" fra gruppen "${groupsById[abakusGroup].name}?"`
-      ) && removeMember(membership);
     return (
       true && (
         <>
-          <i
-            key="icon"
-            className={`fa fa-times ${styles.removeIcon}`}
-            onClick={performRemove}
-          />
+          <ConfirmModalWithParent
+            title={'Bekreft utmelding'}
```

Comment:
`title="str"` instead.

Reply:
Hm, thought prettier would fix this automagicaly. :thinking:

Reply:
Actually, prettier is changing " to ' in this case. I guess this is okay then?

Reply:
Hmm. Sure u tried `title="bar` nad not `title={"bar"}`?

Reply:
Oh, I see. Forgot to remove the curly brackets { }